tests/py_runtime/py_runtime_tests.bzl

```diff
@@ -320,6 +320,32 @@ def _test_system_interpreter_must_be_absolute_impl(env, target):
 
 _tests.append(_test_system_interpreter_must_be_absolute)
 
+def _test_interpreter_sh_binary_target(name):
+    native.sh_binary(
```

Comment:
This needs to use rt_util.helper_target. It just hides the target from `...` expansion so that it isn't built when it shouldn't be (it should only be built as part of the analysis test).

Reply:
Done